crates/scheduler/src/call/crypto/hash.rs

```diff
@@ -154,13 +154,13 @@ fn hmac_update<B: Board>(mut call: SchedulerCall<B, api::hmac_update::Sig>) {
     let api::hmac_update::Params { id, data, length } = call.read();
     let scheduler = call.scheduler();
     let memory = scheduler.applet.store.memory();
-    let result: Result<Result<(), _>, _> = try {
+    let result: Result<(), Failure> = try {
```

Comment:
the whole type annotation can disappear
```suggestion
    let result = try {
```